platform/platform-app/pom.xml

```diff
@@ -455,7 +458,6 @@
         <cxf.xmlresolver.bundle.version>1.2_5</cxf.xmlresolver.bundle.version>
         <cxf.xerces.bundle.version>2.11.0_1</cxf.xerces.bundle.version>
         <cxf.bcel.bundle.version>5.2_4</cxf.bcel.bundle.version>
-        <cxf.xalan.bundle.version>2.7.2_3</cxf.xalan.bundle.version>
```

Comment:
I realize you added this to the root pom but I don't know that we want to remove it from here since this is copied from cxf. I think by keeping them all together here they were trying to make it easier to upgrade next time.